File: Java/Mahasiswa.java
package Java;

public class Mahasiswa {
    
    private String nama, prodi, fakultas;
    private int nim;

    //Konstruktor
    public Mahasiswa() {
        
    }

    /*setter untuk mengisi atau memperbarui nilai variabel*/
    // Set nama
    public void setNama(String nama){
        this.nama = nama;
    }
    // Set prodi
    public void setProdi(String prodi){
        this.prodi = prodi;
    }
    // Set fakultas
    public void setFakultas(String fakultas){
        this.fakultas = fakultas;
    }
    // Set nim
    public void setNim(int nim){
        this.nim = nim;
    }

    /*Getter untuk mengambil nilai variabel*/
    // Get nama
    public String getNama(){
        return this.nama;
    }
    // Get prodi
    public String getProdi(){
        return this.prodi;
    }
    // Get fakultas
    public String getFakultas(){
        return this.fakultas;
    }
    // Get nim
    public int getNim(){
        return this.nim;
    }
}

File: PHP/Mahasiswa.php
<?php

class Mahasiswa{
    private $nama, $prodi, $fakultas, $nim;

    //konstruktor
    public function __construct($nama = "0", $prodi = "0", $fakultas = "0", $nim = "0")
    {
        $this->nim = $nim;
        $this->nama = $nama;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    //Setter
    public function setNim($nim)
    {
        $this->nim = $nim;
    }
    public function setNama($nama)
    {
        $this->nama = $nama;
    }
    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    //Getter
    public function getNim()
    {
        return $this->nim;
    }
    public function getNama()
    {
        return $this->nama;
    }
    public function getProdi()
    {
        return $this->prodi;
    }
    public function getFakultas()
    {
        return $this->fakultas;
    }

    //destruktor
    public function __destruct()
    {
        
    }
}
?>